packages/apps-config/src/api/bundle/spec/moonbeam.ts

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017-2020 @polkadot/apps-config authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { OverrideBundleDefinition } from '@polkadot/types/types';
+
+// structs need to be in order
+/* eslint-disable sort-keys */
+
+export default {
```

Comment:
The format is slightly different here, and probably the issue.

Here is an example,

```export default [
  {
    // lowest version it applies to, highest version it applies to
    minmax: [0, undefined],
    // as per normal
    types: { ... }
  }
]
```

This allos you to manage multiples, see eg. https://github.com/polkadot-js/api/blob/master/packages/types-known/src/spec/kusama.ts where it is done for Kusama

Reply:
thanks :)

However, doesn't that mean that the laminar and acala files use the wrong type (OverrideBundleDefinition)?

Reply:
it looks like using OverrideVersionedType[] like in the example you provided doesn't conform to the type for Api.create of react-api

Reply:
It may be slightly different, honestly would need to look at what it does under the hood. Too many type options. 

(I have been meaning to move all types across for all chains, Acala added the required support into the UI specifically since they had upgrades lines up)

Reply:
@jacogr have you had the occasion to look? Should I try again?

Reply:
Will take a peek and refresh.

Reply:
It should just work as defined here. 

You do `OverrideVersionedType[]` which aligns with types-known, as first used.

When passed to the API, it expects `typesBundle: OverrideBundleType` which is of this form -

```js
typesBundle: {
  spec: {
    moonbeam: OverrideVersionedType[]
  }
}
```

The api-react should take the stuff defined and just put it in that format.

Reply:
Ok, so what you should export is -

```js
export default {
  types: OverrideVersionedType[]
};
```

Reply:
(Like I said once the types is in, we can attempt to move the RPCs as well - not 100% sure the API uses it atm, the intent is there - if it doesn't can fix)

Reply:
Thanks for the help. Should be ready now!

>Like I said once the types is in, we can attempt to move the RPCs as well 

Not too hot on adding features that aren't needed explicitly but I can add that as well if you feel this is important

Reply:
No, I'm sure that stuff is going to break and it is more for my benefit ... so I'll ply with that and see what breaks horribly. However, will get you to review once ready. (Not promising in the next couple of days, but having seen that again makes me want to get it going)